index.html

```diff
@@ -19,10 +20,188 @@ <h1 class="textdeco1 text1width">José Fernández-Caminero</h1>
+        </div>
+
+        <img class="fullWidth" src="resources/Rectangle 21(3).png" alt="Yet another laptop and other objects">
+        <div class="blueBox">
+            <h3 class="projectName">Project name goes here</h3>
+
+            <ul class="greenul">
+                <li class="greenli">HTML/CSS</li>
+                <li class="greenli">Ruby on Rails</li>
+                <li class="greenli">JavaScript</li>
+            </ul>
+
+            <button class="button1 width65 buttonMargin1">See this project
+                <img src="resources/Union.png" alt="Cooler Arrow Pointing Right">
+                <img src="resources/Union2.svg" alt="Cool (Blue) Arrow Pointing Right">
+            </button>
+        </div>
+    
+        <img class="fullWidth" src="resources/Rectangle 21(4).png" alt="Laptop and other objects">
+        <div class="blueBox">
+            <h3 class="projectName">Project name goes here</h3>
+
+            <ul class="greenul">
+                <li class="greenli">HTML/CSS</li>
+                <li class="greenli">Ruby on Rails</li>
+                <li class="greenli">JavaScript</li>
+            </ul>
+
+            <button class="button1 width65 buttonMargin1">See this project
+                <img src="resources/Union.png" alt="An Even Cooler Arrow Pointing Right">
+                <img src="resources/Union2.svg" alt="Cool (Blue) Arrow Pointing Right">
+            </button>
+        </div>
+
+        <div>
+            <img class="fullWidth" src="resources/Rectangle 21(5).png" alt="Another laptop and other objects">
+            <div class="blueBox">
+                <h3 class="projectName">Project name goes here</h3>
+
+                <ul class="greenul">
+                    <li class="greenli">HTML/CSS</li>
+                    <li class="greenli">Ruby on Rails</li>
+                    <li class="greenli">JavaScript</li>
+                </ul>
+
+                <button class="button1 width65 buttonMargin1">See this project
+                    <img src="resources/Union.png" alt="Cool (White) Arrow Pointing Right">
+                    <img src="resources/Union2.svg" alt="Cool (Blue) Arrow Pointing Right">
+                </button>
+            </div>
+        </div>
+
+        <img class="fullWidth" src="resources/Rectangle 21(6).png" alt="Picture of laptop and other objects">
+        <div class="blueBox">
+            <h3 class="projectName">Project name goes here</h3>
+
+            <ul class="greenul">
+                <li class="greenli">HTML/CSS</li>
+                <li class="greenli">Ruby on Rails</li>
+                <li class="greenli">JavaScript</li>
+            </ul>
+
+            <button class="button1 width65 buttonMargin1">See this project
+                <img src="resources/Union.png" alt="Yet An Even Cooler Arrow Pointing Right">
+                <img src="resources/Union2.svg" alt="Cool (Blue) Arrow Pointing Right">
+            </button>
+        </div>
+    </section>
+
+    <header>
```

Comment:
I suggest you change this to a `section` element, the `header` tag represents a container for introductory content while the  `section` element represents a generic standalone section of a document, which in this case is what we are doing semantically 👍 

- According to the project requirement, the  about `section` container should be aligned using flexbox 😉

Reply:
Hi @bimbolabuari 

Thank you so much for such a kind review. I have changed the header tag. You are right, it's semantically incorrect.

However, regarding the About section: on template 5, which is the one I'm working on, it is so far very easy to keep the elements aligned without using flex or grid. Just following the natural flow of the HTML. I'm sure that will change on the Desktop version. Should I use flexbox anyway? Even though it's not required at the moment?

Thanks again.